tests/wamr-test-suites/test_wamr.sh

```diff
@@ -162,8 +162,8 @@ do
         QEMU_FIRMWARE=${OPTARG}
         ;;
         j)
-        echo "test cosmopolitan"
-        PLATFORM="cosmopolitan"
+        echo "test platform" ${OPTARG}
```

Comment:
Had better add a space, "test platform " ${OPTARG}

Reply:
Adjusted spacing to match in 27023ee